csunplugged/topics/content/en/image-representation/image-representation.md

```diff
@@ -0,0 +1,8 @@
+# Image Representation
+
+Images are everywhere on computers and digital devices, and they are one of the main ways we interact with them.
+Digital devices store everything using digits, so how do they use these digits to create the images we see on our computer screens, our smartphones, and our tablets?
+How do these devices convert a long list of digits into photos, video game graphics, and web pages?
+This unit will explore how we can represent pictures and colours using binary digits, how these are displayed using pixels on a screen, and why the way we represent these images changes what we see.
+
+We recommend completing lesson 1 (for the relevant age group) of the [Binary Numbers unit]('topics:unit_plan' 'binary-numbers' 'unit-plan') before beginning this unit.
```

Comment:
Lesson 1 is the name of the lesson, so it is a proper noun and should be capitalised